app/controllers/expenses_controller.rb

```diff
@@ -2,26 +2,28 @@ class ExpensesController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @authorized_expense_user = Expense.joins(:user).where(["expenses.user_id = ? OR expenses.user_id = ? OR users.invited_by_id = ?",
-                                            current_user.id, current_user.invited_by_id, current_user.id])
-    @search = @authorized_expense_user.order(expense_date: 'DESC').ransack(params[:q])
-    @expenses = @search.result.includes(:user).paginate(page: params[:page], per_page: 15)
+    authorized_expense_user = Expense.joins(:user).where(
+        [
+            "expenses.user_id = ? OR expenses.user_id = ? OR users.invited_by_id = ?",
```

Comment:
Line is too long. [86/80]<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.